Formula/mpd.rb

```diff
@@ -39,11 +39,25 @@ class Mpd < Formula
   depends_on "opus"
   depends_on "sqlite"
 
+  uses_from_macos "curl"
+
+  on_linux do
+    depends_on "gcc"
+  end
+
+  fails_with gcc: "5"
+
   def install
     # mpd specifies -std=gnu++0x, but clang appears to try to build
     # that against libstdc++ anyway, which won't work.
     # The build is fine with G++.
-    ENV.libcxx
+    on_macos do
+      ENV.libcxx
```

Comment:
Isn't this a no-op when your compiler isn't `clang`?

Reply:
Probably right.  I'll check locally in Docker and remove the `on_macos` block if it's unneeded.

Reply:
We can also look at [the code](https://rubydoc.brew.sh/Superenv.html#libcxx-instance_method):
```ruby
def libcxx
  append_to_cccfg "g" if compiler == :clang
end
```

Reply:
Not only does `ENV.libcxx` not need to be put inside an `on_macos` block, but we have a fair number of build/test failures in other formulae where `-stdlib=libc++` is added manually, where we should just use `ENV.libcxx` instead.